.devcontainer/devcontainer.json

```diff
@@ -1,22 +1,18 @@
 {
     "name": "Recommenders",
-    // Version list: https://github.com/devcontainers/images/tree/main/src/base-ubuntu
-    // Includes: curl, wget, ca-certificates, git, Oh My Zsh!, 
-    "image": "mcr.microsoft.com/devcontainers/base:ubuntu-24.04",
-    "hostRequirements": {
-        "cpus": 4,
-        "memory": "16gb",
-        "storage": "32gb"
-    },
-    "features": {
-        // https://github.com/devcontainers/features/blob/main/src/anaconda/devcontainer-feature.json
-        "ghcr.io/devcontainers/features/anaconda:1": {
-            "version": "2024.06-1"
+    "build": {
```

Comment:
Some of the GPU tests failed. When looking at the AzureML workspace I see this error: Provisioning error
The specified Virtual Machine size in low priority is currently out of capacity. Please retry later, try reducing the Virtual Machine size or number of instances, try using dedicated VMs to improve chances of capacity allocations, or try deploying to a different region.

I'll try to run the tests again, otherwise, I'll try to see if we need new VMs.

Reply:
I find the VM size is specified in [tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py](https://github.com/recommenders-team/recommenders/blob/397e80afbea49e634655739b8c8f06e913350b84/tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py#L86-L91).  Maybe we need to use another one, because the code was written 2 years ago.

Reply:
The VMs are chosen from the portal. Finally it worked. I think it was a momentary error.
Merging.

Reply:
@SimonYansenZhao the process automatically cancells after 6h. I was trying to find how this limit is defined to reduce it, but I don't know where we are setting this value. Do you know?

Reply:
Just found this: https://stackoverflow.com/a/59076067

<img width="538" alt="Screenshot 2024-11-16 at 09 58 21" src="https://github.com/user-attachments/assets/268f091f-6ed3-4a13-9a03-4e8af6f1f30a">